package-lock.json

```diff
@@ -16508,11 +16507,6 @@
         }
       }
     },
-    "react-bootstrap-switch": {
```

Comment:
Seems like removing this breaks the switch [here](https://176714294.dev.laji.fi/taxon/list?onlyFinnish=true)

Reply:
Good catch, fixed. It's a bit worrisome that changing laji-form styles breaks laji.fi styles... (I removed the bootstrap-switch from laji-form in this PR, and apparently laji.fi wasn't taking care of loading the styles itself. Which makes sense though, because we don't want to include the styles twice)

Reply:
completely agree. Style dependency should have been added to laji.fi side when the switch was added